Problem1.py

```diff
@@ -0,0 +1,15 @@
+# Time Complexity :O(n) where n is length of nums
+# Space Complexity : O(n) for variable n which is a set
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : Yes. in the condition where I have written "if i not in n" if I will write
+# "if i not in set(nums), it threw Time Exceeded error"
```

Comment:
there is no bug in your logic
the error may be because of creating set everytime so if nums is an array of large values that will take alot of time for creating set again and again